OpenRA.Game/Map/Map.cs

```diff
@@ -165,41 +183,65 @@ static object LoadVideos(MiniYaml y)
 			return videos;
 		}
 
-		[FieldLoader.Ignore] public Lazy<Dictionary<string, ActorReference>> Actors;
+		[FieldLoader.Ignore]
```

Comment:
Best practice guidelines often also have a clause that the existing style in a file takes precedence :)
See e.g. point 9 of https://github.com/dotnet/corefx/wiki/Contributing#c-coding-style.

This doesn't mean that we won't change things like this in the future to come more in line with the wider community practices, only that we won't do this kind of thing on a piecemeal basis.  We have been putting a lot of effort recently into automated codestyle verification, so in the future it should be straightforward to identify and fix things like this project-wide if/when we decide it is wanted.